wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md

```diff
@@ -2,142 +2,158 @@
-formed by eigenvectors corresponding to eigenvalue $\lambda$ of a such
-matrix $M$. Let $(t), (t)  $for$t 0$. We will write$(t)  (t)$for
-relatively small$t 0$, whenever $\lambda(0)=\beta(0)$ and $(0)=(0)$.
-
-## Theorem
+# Perturbation of the numerical range of unitary matrices
 
-Let $U \in \mathrm{U}(\mathbb{C}^d)$ be a unitary matrix with spectral
-decomposition  Assume that the eigenvalue $\lambda \in \lambda(U)$ is
-such that $r(\lambda) = k$. Let us define a matrix $V(t)$ given by  Let
-$\lambda(t):=\lambda(UV(t))$ and let every $\_j(t)  (t)$corresponds to
-eigenvector$ $. Assume that $\lambda_{1}(t), \ldots, \lambda_{k}(t)$ are
-such eigenvalues that $\lambda_{j}(t) \to \lambda$, as $t \to 0$. Then:
+Let us consider two unitary matrices - matrix $U \in \mathrm{U}(\mathbb{C}^d)$
+and its perturbation $V \in \mathrm{U}(\mathbb{C}^d)$, such that $\\\|U-V\\\| \le \epsilon$, for some $\epsilon > 0$.
+To connect those unitary matrices, we define geodesic curve $U(t) \in \mathrm{U}(\mathbb{C}^d)$
+for any $t \in \[0,1\]$ with boundary conditions $U(0) \coloneqq U$ and $U(1) \coloneqq V$. 
+This smooth path is explicitly given as<sup>{% cite antezana2014optimal %}</sup> 
+
+\begin{equation}
+t \mapsto U \exp \left( t \text{Log} \left( U^\dagger V \right) \right). 
+\end{equation}
 
-1\. If $*{ S*^U} \_{i=1}^d\\ p\_i ||^2=0$, then $\lambda$ is an
-eigenvalue of $UV(t)$;
+We will study how the numerical range $W(U(t))$ changes depending on $t$. As shown in 
+{% cite kukulski2020 %} we may focus on equivalent, but simplified problem of determining the behavior
+of the numerical range of $U(t) = U \exp \left( \ii t D_+ \right)$, where $D_+$ is diagonal, positive, 
+semi-definite matrix, such that $\tr (D_+) = 1$.
 
-2\. If $\|\\{p_i: p_i\>0\\}\|=l\<k$, then $\lambda$ is an eigenvalue of
-$UV(t)$ and $r(\lambda) \ge k-l$;
+## Theorem<sup>{% cite kukulski2020 %}</sup>
 
-3\. Each eigenvalue of product $UV(t)$ moves counterclockwise or stays
-in the initial position as parameter $t$ increases;
+Let $r(\lambda)$ be an algebraic multiplicity of eigenvalue $\lambda$, define by $I_{U, \lambda}$ the isometry which
+columns are formed by eigenvectors corresponding to eigenvalue $\lambda$ of a matrix $U$ and define
+
+\begin{equation}
+S_{\lambda}^{U} = \left\\{ \ket{x} \in \mathbb{C}^d: (\lambda \1_d - U) \ket{x} = 0, \,\, \\\| x \\\|_2 = 1 \right\\}.
+\end{equation}
+
+Let $U \in \mathrm{U}(\mathbb{C}^d)$ be a unitary matrix with spectral
+decomposition 
 
-4\. If $k=1$, then  for small $t  0$;
+\begin{aligned}
+    U = \sum_{j=1}^d \lambda_j \proj{\lambda_j}
+\end{aligned}
 
-5\. Let $Q:=I_{U,\lambda}^\dagger D_+ I_{U,\lambda}$ and $\lambda_1(Q)
-\le \lambda_2(Q) \le \ldots \le \lambda_k(Q)$. Then we have  for small
-$t \geq 0$ and eigenvector $\ket{ x_j}$ corresponding to $\lambda_j \in
-\lambda(U)$ is given by  where $ S^{Q}\_{\_j(Q)}$;
+Assume that the eigenvalue $\lambda \in \lambda(U)$ is such that $r(\lambda) = k$. Let us define a matrix 
```

Comment:
Why $\lambda(U)$ and not $\lambda \in \{ \lambda_1, \ldots, \lambda_d\}$? If we insist on $\lambda(U)$, then is it defined as a set or a multiset?